whelktool/scripts/cleanups/2019/08/remove-properties-from-auth-records/remove-properties-from-auth-records.groovy

```diff
@@ -0,0 +1,31 @@
+/*
+See LXL-2639 for more info.
+*/
+
+PrintWriter failedIDs = getReportWriter("failed-to-delete-bibIDs")
+PrintWriter scheduledForChange = getReportWriter("scheduledForChange")
+
+selectBySqlWhere("collection = 'auth'", silent: false) { documentItem ->
+    def record = documentItem.doc.data.get('@graph')[0]
+
+    removeFieldFromRecord(record, "marc:subdivision", scheduledForChange)
+    removeFieldFromRecord(record, "marc:languageOfCatalog", scheduledForChange)
+    removeFieldFromRecord(record, "marc:subjectHeading", scheduledForChange)
+    removeFieldFromRecord(record, "marc:headingMain", scheduledForChange)
+    removeFieldFromRecord(record, "marc:headingSubject", scheduledForChange)
+    removeFieldFromRecord(record, "marc:subjectSubdivision", scheduledForChange)
+    removeFieldFromRecord(record, "marc:recordUpdate", scheduledForChange)
+    removeFieldFromRecord(record, "marc:modifiedRecord", scheduledForChange)
+
+    documentItem.scheduleSave(onError: { e ->
+        failedIDs.println("Failed to save ${record[ID]} due to: $e")
+    })
+}
+
+private void removeFieldFromRecord(record, String fieldName, scheduledForChange) {
```

Comment:
Just a nitpick: since the scheduledForChange object is global it doesn't have to be passed.

Reply:
Or actually, since it's typed, it might have had to be? (The behaviour of local methods differs on that.)